qutebrowser/mainwindow/tabwidget.py

```diff
@@ -506,14 +516,6 @@ def refresh(self):
         # code sets layoutDirty so it actually relayouts the tabs.
         self.setIconSize(self.iconSize())
 
-    def _set_font(self):
-        """Set the tab bar font."""
-        self.setFont(config.val.fonts.tabs)
-        self._set_icon_size()
-        # clear tab size cache
-        self._minimum_tab_size_hint_helper.cache_clear()
-        self._minimum_tab_height.cache_clear()
```

Comment:
I suppose this should still be done if the font changes, no?

Reply:
Yes. But this is done in lines 447-448 already (with the addition of the new config values in lines 445-446). Also, `self._set_icon_size()` is now called inside the `_on_config_changed` method. I guess the `_set_font` method could be kept, but all it would do is call `_set_icon_size`.